### PR TITLE
Typo fixes

### LINKED ISSUE #501: [English Typos] File Access Warning and Shortcut
If write access is denied to the user and you try to save a level you will get a warning, it has a typo.

t2

"**can't fopen.**"

How about...

"**Can't open file. Access may be denied or someone else may be saving the same file. Please wait and try again.**"

Users working in environments with rigorous working group policies may be left scratching their head if the project manager has revoked write access for any reason and they get presented with this prompt. The small specification about access may be denied could give some guidance to their issue.

Also our old friend "sensibility" is back under **Shortcut Settings > Tool Modifiers > Pressure _sensibility_**.

Change to:

**Pressure Sensitivity**.

### PR DESCRIPTION
**Pull request by turtletooth**
This fixes #501.

### CODE REVIEW

**Comment by ideasman42** [comment]
Suggest to split into 2 PR's

Also, there are many other typos

```
1	./toonz/sources/include/ext/StrokeParametricDeformer.h:96:*@brief Change sensibility of deformer (just for debug).
2	./toonz/sources/include/ext/StrokeParametricDeformer.h:124:  // sensibility of deformer
3	./toonz/sources/include/toonz/vectorizerparameters.h:40:            This typically includes vectorization modes, various sensibility and
4	./toonz/sources/toonz/mainwindow.cpp:2035:                          tr("Pressure sensibility"), "");
5	./toonz/sources/toonz/trackerpopup.cpp:169:  m_sensibility = new DoubleField();  // W_Accuracy
6	./toonz/sources/toonz/trackerpopup.cpp:170:  m_sensibility->setFixedHeight(WidgetHeight);
7	./toonz/sources/toonz/trackerpopup.cpp:171:  m_sensibility->setRange(0, 1000);
8	./toonz/sources/toonz/trackerpopup.cpp:172:  m_sensibility->setValue(10);
9	./toonz/sources/toonz/trackerpopup.cpp:173:  addWidget(tr("Sensitivity:"), m_sensibility);
10	./toonz/sources/toonz/trackerpopup.cpp:196:  float sensibility    = m_sensibility->getValue() / 1000;
11	./toonz/sources/toonz/trackerpopup.cpp:228:      new Tracker(threshold, sensibility, activeBackground, manageOcclusion,
12	./toonz/sources/toonz/trackerpopup.cpp:285:Tracker::Tracker(double threshold, double sensibility, int activeBackground,
13	./toonz/sources/toonz/trackerpopup.cpp:290:  m_sensibility      = sensibility;
14	./toonz/sources/toonz/trackerpopup.cpp:572:  if ((m_sensibility < 0) || (m_sensibility > 1)) {
15	./toonz/sources/toonz/trackerpopup.cpp:665:    if ((dist_temp < m_sensibility) && (m_pObjectTracker[i]->track)) {
16	./toonz/sources/toonz/trackerpopup.h:49:  DVGui::DoubleField *m_sensibility;
17	./toonz/sources/toonz/trackerpopup.h:88:  float m_sensibility;
18	./toonz/sources/toonz/trackerpopup.h:117:  Tracker(double threshold, double sensibility, int activeBackground,
19	./toonz/sources/toonzlib/sandor_fxs/YOMBInputParam.cpp:145://		25. sensibility of selection (dAB)
20	./toonz/sources/toonzqt/schematicnode.cpp:500:  double sensibility = 5 / scaleFactor;
21	./toonz/sources/toonzqt/schematicnode.cpp:501:  QPointF h(0, sensibility);
22	./toonz/sources/toonzqt/schematicnode.cpp:504:    p.setX(p2.x() > 0 ? -sensibility : sensibility);
23	./toonz/sources/toonzqt/schematicnode.cpp:506:    p.setX(p2.x() > 0 ? sensibility : -sensibility);
```

**Comment by turtletooth** [comment]
@ideasman42 Thanks for pointing out the other typos.  I was just fixing the outward facing ones first, but getting them all at once would be better.

I am still new to github- could you share why splitting these into two PR would be better since they are both related to typos?  I am not trying to be difficult -- just still learning.

**Comment by skitaoka** [comment]
Jenkins, test this.

**Comment by turtletooth** [comment]
Updated to master, capitalized Sensitivity on mainwindow

**Comment by shun-iwasawa** [comment]
Jenkins, again

**Comment by shun-iwasawa** [comment]
Please modify the following part as well, since the tool option command ids (string values starting with "A_ToolOption_") are used for connecting tool properties with corresponding QActions in the function ToolOptionControlBuilder::visit(TBoolProperty *). Thanks! 

```
--------------------- toonz/sources/tnztools/brushtool.cpp ---------------------
@@ -772,7 +772,7 @@ BrushTool::BrushTool(std::string name, int targetType)
   m_prop[0].bind(m_preset);
   m_preset.setId("BrushPreset");
   m_preset.addValue(CUSTOM_WSTR);
-  m_pressure.setId("PressureSensitivity");
+  m_pressure.setId("PressureSensibility");
 
   m_capStyle.addValue(BUTT_WSTR);
   m_capStyle.addValue(ROUNDC_WSTR);
```

**Comment by turtletooth** [comment]
@shun-iwasawa I'm not sure what I should change in your suggestion.  PressureSensitivity is correct and already in there.  If I'm misunderstanding something, I apologize.

File: toonz/sources/toonz/mainwindow.cpp [hunk 1]
```diff
@@ -2031,8 +2031,8 @@ void MainWindow::defineActions() {
   createToolOptionsAction("A_ToolOption_PencilMode", tr("Pencil Mode"), "");
   createToolOptionsAction("A_ToolOption_PreserveThickness",
                           tr("Preserve Thickness"), "");
-  createToolOptionsAction("A_ToolOption_PressureSensitivity",
-                          tr("Pressure sensitivity"), "");
+  createToolOptionsAction("A_ToolOption_PressureSensibility",
```
Comment (shun-iwasawa) [inline toonz/sources/toonz/mainwindow.cpp]: @turtletooth 
Oh, sorry for making you confused! Now I understand. -TIVITY is correct.
Then I think this line 2034 should be kept unchanged. Please forget my previous comment.

**Comment by shun-iwasawa** [comment]
Jenkins

**Comment by shun-iwasawa** [comment]
LGTM

**Pull request merged**